spec/support/shared/availability_check.rb

```diff
@@ -140,44 +254,99 @@
 
   context "when there is an application" do
     context "when it is available" do
-      it "updates the availability status to available" do
-        # GET
-        stub_not_found(:endpoint)
-        stub_get(:application, list_applications_response)
-        # PATCH
-        application_patch_body = {'last_available_at' => subject.send(:check_time), 'last_checked_at' => subject.send(:check_time)}.to_json
-        source_patch_body = {'availability_status' => described_class::STATUS_AVAILABLE, 'last_available_at' => subject.send(:check_time), 'last_checked_at' => subject.send(:check_time)}.to_json
+      context 'kafka' do
+        it "updates the availability status to available" do
+          stub_not_found(:endpoint)
+          stub_get(:application, list_applications_response)
 
-        stub_patch(:source, source_patch_body)
-        stub_patch(:application, application_patch_body)
+          error_message = 'error_message'
```

Comment:
The same like ^^, string 'error_message' is confusing when status is 'available'